src/Dialog/DialogCentreScreen.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import styled from "styled-components";
+import {
+  useDialogState,
+  Dialog,
+  DialogBackdrop,
+  DialogOptions
+} from "reakit/Dialog";
+import { colors } from "../shared/constants";
+
+interface DialogProps {
+  label: string;
+  dialog?: DialogOptions;
+  [key: string]: any;
+}
+
+const StyledBackdrop = styled(DialogBackdrop)`
+  box-sizing: border-box;
+  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+  background-color: rgba(0, 0, 0, 0.5);
+  position: fixed;
+  top: 0px;
+  right: 0px;
+  bottom: 0px;
+  left: 0px;
+  z-index: 999;
+`;
+
+const StyledDialog = styled(Dialog)`
+  display: flex;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  -webkit-transform: translate(-50%, -50%);
+  -ms-transform: translate(-50%, -50%);
```

Comment:
`styled-components` library automatically creates these vendor specific style definitions when it renders. No need to add them.